pkg/storage/imagestore/imagestore.go

```diff
@@ -1572,10 +1598,7 @@ func (is *ImageStore) CleanupRepo(repo string, blobs []godigest.Digest, removeRe
 		}
 	}
 
-	blobUploads, err := is.storeDriver.List(path.Join(is.RootDir(), repo, storageConstants.BlobUploadDir))
-	if err != nil {
-		is.log.Debug().Str("repository", repo).Msg("failed to list .uploads/ dir")
-	}
+	blobUploads, _ := is.GetBlobUploads(repo)
```

Comment:
Because it was already ignored before. The in the old code was only logging the error, and that logic is moved in the new function.

There is no reason to stop the repo cleaning logic (which removes the repo at line 1607) just because of this error.